homeassistant/components/alexa/handlers.py

```diff
@@ -924,3 +943,53 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return response
+
+
+@HANDLERS.register(('Alexa.ChannelController', 'ChangeChannel'))
+async def async_api_changechannel(hass, config, directive, context):
+    """Process a change channel request."""
+    channel = 0
+    entity = directive.entity
+
+    if 'number' in directive.payload['channel'].keys():
+        channel = directive.payload['channel']['number']
+    elif 'callSign' in directive.payload['channel'].keys():
+        channel = directive.payload['channel']['callSign']
+    elif 'name' in directive.payload['channelMetadata'].keys():
+        channel = directive.payload['channelMetadata']['name']
+
```

Comment:
According the Alexa API `channel.number`, `channelMetadata.name`, `channel.callSign`, `channel.affiliateCallSign` or `channel.uri` will be provided in the directive. Is it possible to handle the `channel.affiliateCallSign` and `channel.uri` too?